cmake/modules/FindUFE.cmake

```diff
@@ -82,6 +83,13 @@ find_library(UFE_LIBRARY
     NO_DEFAULT_PATH
 )
 
+# Can not be FALSE or 0 as this implies "not found" and fails.
+set(UFE_PREVIEW_BATCHOPS_SUPPORT "absent")
```

Comment:
I now see why CACHED variables are used for bolean CMake variables. Here is an alternative that works,.

@seando-adsk : If you find that style of CMake unacceptable, I have no issue reverting to a CACHED boolean CMake variable.